pkg/types/binary_test.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+)
+
+func TestBinary_MarshalJSON(t *testing.T) {
```

Comment:
Please use `PascalCase` and `camelCase` as test names.

Reply:
Goland suggested these. So they must be a convention or something.

Reply:
Wait! I have a hint from the IDE, at least for the helpers to use camel case instead of snake case. @julianbrost

Reply:
Typically, you don't use `_` in function names in Go. But test function names seem to be a quite common exception:
```console
% grep -r 'func Test[^(]*_' "$GOPATH" | wc -l 
10014
```

Reply:
I was referring to the auxiliary functions:
https://github.com/Icinga/icingadb/blob/b0f2aa3c42bac1651e020e404b08e65655dd1ae0/pkg/types/binary_test.go#L15-L16

Reply:
Personally, I think `_` looks strange in function names in Go, but that's very subjective. Anyways, I think this discussion is now irrelevant as that was changed already.